### PR TITLE
Adding Timing for Guided, Loiter, and Stabilization Control Loops

### PR DESCRIPTION
**Pull request by jburns11**
Here I added the code to printout timing statistics  of the 3 control loops mentioned above. This is set through the public static boolean "LOG_TIMING", located in Copter.java. I also made a small edit to the make file, and readme. These were just minor things I ran into when trying to install.

### CODE REVIEW

**Comment by adamczer** [comment]
resolve conflicts and retest, i tried to resolve a bit but i think something is a little off. Im seeing the below exception, looks like you made a couple changes yesterday that may have messed up what you were intending to call.

Exception in thread "main" java.lang.IllegalStateException: Wrong call, should be using underling implementation not this class.
	at ub.cse.juav.copter.modes.Mode.run(Mode.java:56)

File: juav-copter/src/main/java/ub/cse/juav/copter/Copter.java [hunk 1]
```diff
@@ -31,7 +32,12 @@ void update_flight_mode()
 
         int mode = getModeNumber();
         if(modes.containsKey(mode)) {
-            modes.get(mode).run();
+            long time1 = System.nanoTime();
+            modes.get(mode).run(LOG_TIMING);
```
Comment (adamczer) [inline juav-copter/src/main/java/ub/cse/juav/copter/Copter.java:36]: this is the line that calls the function that throws the exception

**Comment by adamczer** [comment]
make sure to pull again, I have added the required fiji jars to the repo in the event you didnt get a chance to set up fiji yet

**Comment by jburns11** [comment]
Hey I just fixed the error you were referring to. I still had the LOG_TIMING being passed as an argument, as that was my first solution

**Pull request merged**